packages/vite/src/client/overlay.ts

```diff
@@ -71,6 +71,16 @@ pre::-webkit-scrollbar {
   display: none;
 }
 
+pre.frame::-webkit-scrollbar {
+  display: block;
+  height: 5px;
+}
+
+pre.frame::-webkit-scrollbar-thumb {
+  background: #999;
+  border-radius: 5px;
+}
```

Comment:
I think we need the following CSS for Firefox.
```css
pre.frame {
  scrollbar-width: thin;
}
```

Reply:
Ok, updated.